cypress/e2e/v1.0/accommodation/questions/accommodationStatus.ts

```diff
@@ -0,0 +1,223 @@
+export default (stepUrl: string, summaryPage: string, positionNumber: number) => {
+  const question = "What is Paul's current accommodation?"
+
+  describe(question, () => {
+    const options = {
+      settled: 'Settled',
+      temporary: 'Temporary',
+      noAccommodation: 'No accommodation',
+    }
+
+    it('displays and validates the question', () => {
+      cy.getQuestion(question)
+        .isQuestionNumber(positionNumber)
+        // .hasHint('') TODO: Update to test for HTML hint?
+        .hasRadios([options.settled, options.temporary, options.noAccommodation])
+
+      cy.saveAndContinue()
+
+      cy.assertStepUrlIs(stepUrl)
+      cy.getQuestion(question).hasValidationError('Select current accommodation')
+    })
+
+    const typesOfSettledAccommodation = [
+      'Homeowner',
+      'Living with friends or family',
+      'Renting privately',
+      'Renting from social, local authority or other',
+      'Residential healthcare',
+      'Supported accommodation',
+    ]
+
+    it(`displays and validates the conditional options for ${options.settled}`, () => {
+      cy.getQuestion(question).getRadio(options.settled).hasConditionalQuestion(false).clickLabel()
+      cy.getQuestion(question).getRadio(options.settled).getConditionalQuestion().hasRadios(typesOfSettledAccommodation)
+
+      cy.saveAndContinue()
+
+      cy.assertStepUrlIs(stepUrl)
+      cy.getQuestion(question)
+        .getRadio(options.settled)
+        .getConditionalQuestion()
+        .hasValidationError('Select the type of settled accommodation')
+    })
+
+    typesOfSettledAccommodation.forEach(typeOfSettledAccommodation => {
+      it(`summary page displays "${options.settled} - ${typeOfSettledAccommodation}"`, () => {
+        cy.visitStep(stepUrl)
+        cy.getQuestion(question).getRadio(options.settled).clickLabel()
+        cy.getQuestion(question)
+          .getRadio(options.settled)
+          .getConditionalQuestion()
+          .getRadio(typeOfSettledAccommodation)
+          .clickLabel()
+
+        cy.saveAndContinue()
+
+        cy.visitStep(summaryPage)
+        cy.getSummary(question).getAnswer(options.settled).hasSecondaryAnswer(typeOfSettledAccommodation)
```

Comment:
Be good to add these three lines to every test that goes through to the summary page:

```suggestion
        cy.getSummary(question).getAnswer(options.settled).hasSecondaryAnswer(typeOfSettledAccommodation)
cy.getSummary(question).clickChange()
cy.assertStepUrlIs(stepUrl)
cy.assertQuestionUrl(question)
```